subset/security/brute_test/src/main/java/SshSocket.java

```diff
@@ -0,0 +1,138 @@
+import com.google.gson.Gson;
+import com.google.gson.JsonIOException;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonSyntaxException;
+import com.jcraft.jsch.Channel;
+import com.jcraft.jsch.JSch;
+import com.jcraft.jsch.JSchException;
+import com.jcraft.jsch.Session;
+
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.Map;
+
+public class SshSocket implements Runnable {
+
+  String host;
+  int connectionPort;
+  Channel channel;
+  Session session;
+  JSch jsch = new JSch();
+  int attempts = -1;
+  int passwordIndex = 0;
+  int usernameIndex = 0;
+  Report reportHandler;
+  Map macDevices;
+  boolean testFinished = false;
+
+  InputStream jsonStream = this.getClass().getResourceAsStream("/defaultPasswords.json");
+  String testName = "security.passwords";
+  String[] jsonUsernames;
+  String[] jsonPasswords;
+  Gson gson = new Gson();
+  boolean macRetrieved = false;
+  String macAddress;
+  String formattedMac;
+
+  public SshSocket(String host, Map macDevices, int connectionPort, String macAddress) {
+    this.macDevices = macDevices;
+    this.connectionPort = connectionPort;
+    this.host = host;
+    this.macAddress = macAddress;
+    reportHandler = new Report();
+  }
+
+  private void getMACAddress() {
+    try {
+      macAddress = macAddress.replace(":", "");
+      formattedMac = macAddress.substring(0, 6).toUpperCase();
+      System.out.println("MAC ADDRESS : " + macAddress + "  " + macDevices.get(formattedMac));
+      getJsonFile(formattedMac);
+    } catch (Exception e) {
+      System.out.println(e);
+      Report reportHandler = new Report();
+      reportHandler.addText("RESULT skip security.passwords");

```

Comment:
Do we also need a reason the test failed as in the example in bacnet.version here?

https://github.com/JoelRedstone/daq/blob/978a164cf47e30b4c24a2ec90741be1e4a2a07d7/subset/bacnet/bacnetTests/src/main/java/VersionTest.java#L28

Reply:
Have a format for that new line to add to the report? 
E.g. 
<testname> Test
RESULT <result> <testname>
INFO <reason>